### DISCUSSION TITLE
GPT-4 model not found

### DISCUSSION THREAD

**Discussion by Waaaghlace** [Zero6992/chatGPT-discord-bot#359] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/129057404/233346684-d121fb33-9472-4425-ad7a-16b1c9bec3b9.png)

It says "The model: `gpt-4` does not exist", how can I fix it ?

**Comment by Zero6992** [comment, marked as answer]
Your account must be able to access gpt-4
 
Apply here：https://openai.com/waitlist/gpt-4-api
